packages/legacy-client/src/client/OsdkLegacyObject.test.ts

```diff
@@ -22,9 +22,12 @@ import type { OsdkLegacyObjectFrom } from "./OsdkLegacyObject";
 describe("OsdkLegacyObject", () => {
   it("compiles", async () => {
     const x: Todo = {} as Todo;
-    const y: OsdkLegacyObjectFrom<typeof MockOntology, "Todo"> = x;
+    const y: OsdkLegacyObjectFrom<MockOntology, "Todo"> = x;
```

Comment:
I don't think that this is actually an example of a breaking change, but better to flag it than miss it.

Reply:
that mock is not generated, its old. this shouldnt be an issue for those using generated.